src/logger.ts

```diff
@@ -0,0 +1,18 @@
+import winston from "winston";
+import { ctx } from "./ctx";
+
+export const logger = winston.createLogger({
+	level: ctx.logLevel,
+	format: winston.format.combine(
+		winston.format.splat(),
+		winston.format.timestamp({ format: "YYYY-MM-DD HH:mm:ss" }),
+		winston.format.printf((info) => {
+			return `[${info.timestamp}] ${info.level}: ${info.message}`;
+		}),
+		winston.format.colorize({ all: true })
+	),
+	transports: [
+		new winston.transports.File({ filename: "logs/errorTs.log", level: "error" }),
```

Comment:
Unnecessary Ts suffix just make that a `error.log`
Also why only errors?

Reply:
whoops, thats copy paste code :D